src/collect/cli.rs

```diff
@@ -178,6 +180,7 @@ impl SubCommand for Collect {
             .collectors
             .command()
             .to_owned()
+            .alias("c")
```

Comment:
What about:

```diff
modified   src/cli/cli.rs
@@ -265,7 +265,8 @@ impl ThinCli {
         let args: Vec<OsString> = args.into_iter().map(|x| x.into()).collect();
         let mut command = MainConfig::augment_args(Command::new("retis"))
             .version(version)
-            .subcommand_required(true);
+            .subcommand_required(true)
+            .infer_subcommands(true);
         // Add thin subcommands so that the main help shows them.
         for (_, sub) in self.subcommands.iter() {
             command = command.subcommand(sub.thin().expect("thin command failed"));
```

should be more flexible and behaves similarly to iproute2